tools/RAiDER/statsPlot.py

```diff
@@ -351,6 +369,11 @@ def resid(x, d, v, m):
                                    loss='soft_l1', f_scale=0.1,
                                    args=(d, v, model))
 
+        # convert range to specified output unit
+        # TODO temp fix first multiply by 1,010,000 m as crude conversion from degrees to km
+        res_robust.x[0] = res_robust.x[0]*1010000
+        res_robust.x[0] = convert_SI(res_robust.x[0], 'm', 'km')
+
         with warnings.catch_warnings():
```

Comment:
Probably the easiest would be that you just compute distance and multiply it there with 110km per degree so you do that change once.

Reply:
Figured out how to change the distance once. The bug I experience yesterday is gone.